app/domain/transfer/usecase/get_by_account_id.go

```diff
@@ -5,8 +5,14 @@ import (
 )
 
 func (t Transfer) GetTransfersByAccountID(accountID string) ([]entities.Transfer, error) {
+	t.logger.Info().
+		Str("account_ID", accountID).
+		Msg("getting transfers for account id")
 	transfers, err := t.transferRepository.GetTransfersByAccountID(accountID)
 	if err != nil {
+		t.logger.Error().Err(err).
+			Str("account_ID", accountID).
+			Msg("error occurred when was trying to get transfer for account id")
```

Comment:
gonna take this as an example, but it seems there's a code replication here. Maybe you could do something like:
```suggestion
        log := t.logger.With()
        	Str("account_ID", accountID).
		Logger()
		
        log.Info().
		Msg("getting transfers for account id")
	transfers, err := t.transferRepository.GetTransfersByAccountID(accountID)
	if err != nil {
		log.Error().Err(err).
			Msg("error occurred when was trying to get transfer for account id")
```